packages/core/core/src/Transformation.js

```diff
@@ -714,22 +724,52 @@ export default class Transformation {
   }
 
   async loadTransformerConfig(
-    transformer: LoadedPlugin<Transformer<mixed>>,
+    transformer: LoadedPlugin<Transformer<mixed, mixed>>,
     isSource: boolean,
   ): Promise<?Config> {
+    // Only load global config for a transformer once per build.
+    let globalConfig = globalConfigs.get(transformer.name);
+    if (globalConfig == null && transformer.plugin.loadGlobalConfig != null) {
+      globalConfig = createConfig({
+        plugin: transformer.name,
+        searchPath: toProjectPathUnsafe('index'),
+        // Project root should be considered source?
+        isSource: true,
+      });
+
+      await loadPluginConfig(
+        transformer.name,
+        transformer.plugin.loadGlobalConfig,
+        globalConfig,
+        this.options,
+      );
+
+      globalConfigs.set(transformer.name, globalConfig);
+    }
+
+    if (globalConfig) {
+      // I believe this should be done per file?
+      for (let devDep of globalConfig.devDeps) {
+        await this.addDevDependency(devDep);
```

Comment:
I assumed I should be adding dev deps for all files using the global config even though it's already loaded. Just confirming this is correct?